cmd/porter/credentials.go

```diff
@@ -84,6 +84,8 @@ will then provide it to the bundle in the correct location. `,
 	f := cmd.Flags()
 	f.StringVarP(&opts.Namespace, "namespace", "n", "",
 		"Namespace in which the credential set is defined. Defaults to the global namespace.")
+	f.StringSliceVarP(&opts.Labels, "label", "l", nil,
```

Comment:
Is this on the `credentials generate` command?  Should be add a corresponding example in the cmd `Example` section?

Perhaps similar for `parameters generate`, too, if applicable.